src/QuadForm/Herm/GenusRep.jl

```diff
@@ -427,6 +427,31 @@ end
 #
 ################################################################################
 
+# Given a degree 2 extension of number field E/K, with Eabs an absolute simple
+# number field isomorphic to E and R a maximal order in E, return the quotient
+# C/C_0 where C is the class group of E and C_0 is the subgroup of classes
+# consisting of s-invariant ideals. C_0 is generated by the ramified
+# primes in E, and the representatives in the class group of K.
+#
+# We denote by s the generator of Gal(E/K)
```

Comment:
```suggestion
"""
Given a degree 2 extension of number field E/K, with Eabs an absolute simple
number field isomorphic to E and R a maximal order in E, return the quotient
C/C_0 where C is the class group of E and C_0 is the subgroup of classes
consisting of s-invariant ideals. C_0 is generated by the ramified
primes in E, and the representatives in the class group of K.
We denote by s the generator of Gal(E/K)
"""
```
I prefer a docstring over a comment, because like that I can find it in the repl.